src/main/java/seedu/multitasky/logic/parser/ListCommandParser.java

```diff
@@ -25,28 +29,81 @@ public ListCommand parse(String args) throws ParseException {
         if (trimmedArgs.isEmpty()) {
             return new ListCommand();
         }
-
-        // Flags are delimited by whitespace
-        final String[] flags = trimmedArgs.split("\\s+");
-        if (!hasValidFlagCombination(flags)) {
+        // Preamble is necessary due to how ArgumentTokenizer works
```

Comment:
hmm I think I'll leave this here for now, maybe in the next iteration we can do a small change to tokenizer to accept args with and without a preamble